sdk/python/jobs/pipelines/2a_train_mnist_with_tensorflow/README.md

```diff
@@ -7,7 +7,7 @@ products:
 description: This sample shows how to run build pipeline with command function.
```

Comment:
nit: Not sure if unique descriptions matter for indexing, but this one doesn't really fit